config.xml

```diff
@@ -28,9 +28,13 @@
         <icon density="xhdpi" src="res/android/xhdpi/icon.png" />
         <icon density="xxhdpi" src="res/android/xxhdpi/icon.png" />
         <icon density="xxxhdpi" src="res/android/xxxhdpi/icon.png" />
+        <edit-config file="app/src/main/AndroidManifest.xml" mode="merge" target="/manifest/application" xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
Do we need the `xmlns:android` attribute here?

Reply:
Also, I think it was breaking for me because I was trying to do this in the plugin rather than the main configuration file, which would be ideal but I think that's the broken section here.